Cargo.toml

```diff
@@ -21,6 +21,7 @@ resolver = "2"
 
 [patch.crates-io]
 xtra = { git = "https://github.com/comit-network/xtra" } # We need to use unreleased patches.
-maia = { git = "https://github.com/comit-network/maia" } # Unreleased
-maia-core = { git = "https://github.com/comit-network/maia" } # Unreleased
+maia = { git = "https://github.com/comit-network/maia", rev = "cf41b907e6eb68245dcb7c561ec7d4e6bcc4ed19" } # FIXME: point back to master after https://github.com/comit-network/maia/pull/36 gets merged
+
+maia-core = { git = "https://github.com/comit-network/maia", rev = "e419cb2cf19bed6ec53eaa3672408a6205a9b705", package = "maia-core" } # Unreleased version, pinned before maia's breaking change
```

Comment:
🙃 `Unreleased version` this will get outdated as soon as we release. Not sure this helps. 

We can assume that `0.4.19` will be the release. Maybe just hardcode the version here...?

Reply:
the "Unreleased" comment refers to `maia`, not itchysats!